### PR TITLE
Adds legend to pharmacy pages

### PR DESCRIPTION
**Pull request by ecerta**
To test:

- do all requisite vagrant stuff to get you to the right repository
- gulp build
- head over to [the pdf page](http://vitalschoice.dev/pdf/#/)
- Scroll down to a page with pharmacies (page 108 by my count)
- Be filled with awe as you see that each pharmacy now has a teeny tiny letter underneath it indicating what kind of pharmacy it is.
- Marvel and be amazed at the legend that now appears at the bottom of the page. Should look like this:

screen shot 2015-11-19 at 3 08 55 pm

### CODE REVIEW

File: src/client/app/pdf/pdf.service.js [hunk 1]
```diff
@@ -191,6 +189,10 @@
             // Push locations to page 
             pages[pageIndex].locations.push(_location);
             pages[pageIndex].disclaimers = matchDisclaimer(provider_type);
+
+            if (provider_type === 'Pharmacies'){
+              pages[pageIndex].legends = config.legends['Pharmacies'].text;
```
Comment (natemcdowel) [inline src/client/app/pdf/pdf.service.js]: Need to check if config.legends.Pharmacies is set

File: src/client/json/client_config/gphlth.json [hunk 2]
```diff
@@ -9,5 +9,10 @@
 		"Pharmacies":{
```
Comment (robincurry) [inline src/client/json/client_config/gphlth.json]: Should this file live over with the rest of our client configuration here?: https://github.com/mdx-dev/platform-provisioning/tree/next/modules/platform/files/clients

Comment (natemcdowel) [inline src/client/json/client_config/gphlth.json]: If that is how the rest of the platform-* apps are handling config, that seems reasonable.  I think we'd want to work together to figure out how to pull the config file from platform-provisioning, into this app.  Is there an example of this being done in another repo?

Comment (robincurry) [inline src/client/json/client_config/gphlth.json]: If you put a file inside https://github.com/mdx-dev/platform-provisioning/tree/next/modules/platform/files/clients/gphlth, I *think* plat-config will drop it inside the config folder for the application at build time.

Comment (mrchucho) [inline src/client/json/client_config/gphlth.json]: @robincurry it's still white-listed. We could either remove the whitelist of add this new file. But that'd be the way to go @natemcdowel.

**Comment by natemcdowel** [comment]
Thanks for the feedback guys.  I will ping @mrchucho about getting the config integrated with platform-provisioning early next week.

**Pull request merged**